docs/en/usage-with-symfony-console.md

```diff
@@ -61,7 +61,7 @@ final class MigrationContainer
                 'updateNamespaces()' => [['Yii\\User\\Framework\\Migration']],
                 'updatePaths()' => [[]],
             ],
-            MigrationInformerInterface::class => NullMigrationInformer::class,
+            MigrationInformerInterface::class => ConsoleMigrationInformer::class,
```

Comment:
To need to fix in configuration also: https://github.com/yiisoft/db-migration/blob/master/config/di-console.php#L20